scripts/run_country_incid_crop.R

```diff
@@ -140,17 +139,17 @@ if (Sys.getenv("RUN_ON_MARCC", FALSE)) {
 
   #======Use the ocvImpact package======#
   if (!require('ocvImpact', character.only = T)) {
-    roxygen2::roxygenise("packages/ocvImpact")
```

Comment:
I think we just want to remove this, right? I think it was causing problems when running array jobs on idmodeling, right?